content/api/tracking-domains.apib

```diff
@@ -354,10 +354,15 @@ Delete an existing tracking domain.
 
 
 
-### List All Tracking Domains [GET /v1/tracking-domains]
+### List All Tracking Domains [GET /v1/tracking-domains{?default,subaccounts,delimiter}]
 
 Returns an array with all tracking domains. Each item in the array is a full tracking domain object.
 
++ Parameters
+    + default = `false` (optional, boolean) ... Restrict results to default domains when true.
+    + subaccounts (optional, list) ... Delimited list of subaccounts to include in results.
+    + delimiter = `,` (optional, string) ... Specifies the delimiter for query parameter lists.
```

Comment:
I guess we do this other places?  not sure we need it for a control endpoint?  (do we do this for other control endpoints?)

Reply:
This isn't a control endpoint

Reply:
oh right, it's public, but really for internal use

Reply:
Having a query parameter to specify the delimiter to use in another query parameter seems a bit odd. Do we really need it?

Clients can URI-encode "," as %2c in the query parameter. That should be automatically-decoded by Express as part of it's URL decoding? And client libraries should automatically URI-encode query parameters.

I think we can just define it as a comma-separated list.

Reply:
I based the query parameters on the message events api. 'delimiter' is a parameter there